firmware/src/main.rs

```diff
@@ -1,3 +1,9 @@
+use ic_agent::{identity::Secp256k1Identity, Identity};
+use k256::SecretKey;
+
 fn main() {
-    println!("Hello, world!");
+    let secret_key = include_bytes!("../secret_key");
```

Comment:
How do I get this file?

Reply:
When gateway sees new firmware request from
Canister, it generates this file and builds new firmware with it.